third-party/cmake/BuildTSParsers.cmake

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
`BuildTreesitterParsers.cmake`

(I think `ts` is good as a prefix or pseudo-namespace when we need it, but for files like this it seems obscure, and may help with grepping. )